docs/getting-started.rst

```diff
@@ -451,7 +462,7 @@ command run via ``cron``. If we wanted to write a bot, too, we could put its
 listening logic inside the ``chat-messages`` consumer, as every message would
 pass through it.
 
-Linearization
+Linearisation
```

Comment:
I've changed my mind on UK vs US English - if this is going to be merged into Django, it needs to use US English, so probably need to keep the z. Sorry!